custom_components/hubitat/fan.py

```diff
@@ -0,0 +1,76 @@
+"""Support for Hubitat fans."""
+
+from logging import getLogger
+import re
+from typing import Any, List, Optional
+
+from hubitatmaker import (
+    ATTR_SWITCH,
+    ATTR_SPEED,
+    CAP_FAN_CONTROL,
+    CMD_SET_LEVEL,
+    CMD_OFF, 
+    CMD_ON, 
+    CMD_CYCLE_SPEED, 
+    CMD_SET_SPEED,
+    DEFAULT_FAN_SPEEDS,
+    Device
+)
+
+from homeassistant.components.fan import (
+    FanEntity
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.util import color as color_util
+
+from .device import Hub, HubitatEntity, get_hub
+_LOGGER = getLogger(__name__)
+
+class HubitatFan(HubitatEntity, FanEntity):
+    """Representation of a Hubitat fan."""
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if the switch is on."""
+        return self.get_str_attr(ATTR_SWITCH) == "on"
```

Comment:
```suggestion
         if CAP_SWITCH in self._device.capabilities:
             return self.get_str_attr(ATTR_SWITCH) == "on"
         return super().is_on
```

[Hubitat fans](https://docs.hubitat.com/index.php?title=Driver_Capability_List#FanControl) aren't necessarily switches, so we shouldn't assume they have "off" and "on" commands.